### PR TITLE
MM-31047 - Add 10 second timer to list in devMode && testingMode

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Decided 10 seconds was more Prapti-friendly than 1 minute (quicker iteration, no reason to wait around).

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-31047

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]

**Review by crspeller — APPROVED** [review]
Just one nit.

File: server/config/service.go [hunk 1]
```diff
@@ -160,3 +160,19 @@ func (c *ServiceImpl) setConfiguration(configuration *Configuration) {
 
 	c.configuration = configuration
 }
+
+// IsConfiguredForDevelopmentAndTesting returns true when the server has `EnableDeveloper` and
+// `EnableTesting` configuration settings enabled.
+func (c *ServiceImpl) IsConfiguredForDevelopmentAndTesting() bool {
+	config := c.api.Configuration.GetConfig()
+
+	if config != nil &&
```
[the remark this answers is not in the document]
Comment (cpoile) [reply server/config/service.go]: Good idea

**Comment by cpoile** [comment]
e2e test will be fixed when #397 is merged

**Pull request merged**